core/common/src/test/java/alluxio/security/group/GroupMappingServiceTest.java

```diff
@@ -11,35 +11,35 @@
 
 package alluxio.security.group;
 
-import alluxio.Configuration;
-import alluxio.ConfigurationTestUtils;
+import alluxio.ConfigurationRule;
 import alluxio.PropertyKey;
 import alluxio.security.group.provider.IdentityUserGroupsMapping;
 
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
 
 /**
  * Unit test for {@link alluxio.security.group.GroupMappingService}.
  */
 public final class GroupMappingServiceTest {
 
+  @Rule
+  public ConfigurationRule mConfigurationRule = new ConfigurationRule(
+      PropertyKey.SECURITY_GROUP_MAPPING_CLASS, IdentityUserGroupsMapping.class.getName());
+
   /**
    * Tests the {@link GroupMappingService#getGroups(String)} method.
    */
   @Test
   public void group() throws Throwable {
     String userName = "alluxio-user1";
 
-    Configuration.set(PropertyKey.SECURITY_GROUP_MAPPING_CLASS,
```

Comment:
@cmuyaol 

for this particular case, instead of changing the configuration for entire test suite, you can make the following change to apply change on `Configuration` Singleton for a single test case:

```java

try (Closable = 
    new ConfigurationRule(PropertyKey.SECURITY_GROUP_MAPPING_CLASS, 
        IdentityUserGroupsMapping.class.getName()).toResource()) {
  GroupMappingService groups = GroupMappingService.Factory.get();
 
  Assert.assertNotNull(groups);
  Assert.assertNotNull(groups.getGroups(userName));
  Assert.assertEquals(groups.getGroups(userName).size(), 1);
  Assert.assertEquals(groups.getGroups(userName).get(0), userName);
}
```
The advantage of above pattern is that the `Configuration` singleton is set when entering the `try` block and reset when leaving the `try` block